ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -473,7 +488,10 @@ module Vendor_nvidia = struct
             psubdev_id;
             vdev_id = int_of_string (get_attr "deviceId" devid);
             vsubdev_id = int_of_string (get_attr "subsystemId" devid);
-            sriov = sriov
+            sriov = match !Xapi_globs.nvidia_t4_sriov with
+              | Xapi_globs.Nvidia_DEFAULT  -> sriov (* from XML *)
+              | Xapi_globs.Nvidia_T4_SRIOV -> is_sriov id (* true if T4 card *)
+              | Xapi_globs.Nvidia_LEGACY   -> false (* don't use SRIOV *)
```

Comment:
Should the sriov status be logged at debug level?
Perhaps all 3 (`sriov` from xml, `is_sriov id` and the final `sriov`) to help debug issues with the xml/drivers.

Reply:
The configuration flag is always logged; currently not logged is what each card is using. Could log is as well because it is indeed difficult so see.